client-mvp-04/package.json

```diff
@@ -37,7 +37,9 @@
     "build": "react-scripts build",
     "test": "react-scripts test --watchAll=false",
     "test:watch": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "lint": "eslint ./**/*.{js,jsx}",
```

Comment:
Would this include the node_modules folder in the same directory? should we change the glob to exclude node_modules? or just run eslint on src dir?
```json
"eslint \".src/**/*.{js,jsx}\""
```
Not sure about this one

Reply:
Yes. I had also that question in my mind. But it doesn't run on `node_modules` folder.

Reply:
> just run eslint on src dir?

Yes, it can do. I think in the future **js/jsx** file might have outside of the `src` folder. So in that case it will not detect any error from those files, if we only check the `src` folder. I have made it like this by thinking about future possibilities.

Reply:
And the main goal was not to check errors from all files by running the command for this issue. I had added it because I had not found any problem with the config before making the PR. This issue's goal is just to make sure is `eslint` is working the same for everyone. So, let's first try to fix the main problem & get the opinions from other developers at the end about what to do.